packages/client/src/runtime/core/engines/common/types/Transaction.ts

```diff
@@ -1,16 +1,4 @@
-export enum IsolationLevel {
-  ReadUncommitted = 'ReadUncommitted',
-  ReadCommitted = 'ReadCommitted',
-  RepeatableRead = 'RepeatableRead',
-  Snapshot = 'Snapshot',
-  Serializable = 'Serializable',
-}
-
-export type Options = {
-  maxWait?: number
-  timeout?: number
-  isolationLevel?: IsolationLevel
-}
+export { IsolationLevel, type TransactionOptions as Options } from '@prisma/client-engine-runtime'
```

Comment:
I didn't want to extract a new package just for these, and I can't put them in `@prisma/internals` because I want to avoid using it in `@prisma/client-engine-runtime`.